pyproject.toml

```diff
@@ -34,6 +34,7 @@ description = "Home Assistant cloud integration by Nabu Casa, Inc."
 license = {text = "GPL v3"}
 name = "hass-nabucasa"
 readme = "README.md"
+repository = "https://github.com/NabuCasa/hass-nabucasa"
```

Comment:
This should be under `project.urls` (https://packaging.python.org/en/latest/specifications/well-known-project-urls/#well-known-project-urls)

Like:
```
[project.urls]
Repository = "https://github.com/NabuCasa/hass-nabucasa.git"
```